product_supplierinfo_qty_multiplier/models/product_supplierinfo.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 FactorLibre - Hugo Santos
+# Copyright 2021 Tecnativa - Víctor Martínez
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ProductSupplierinfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    multiplier_qty = fields.Float(string="Multiplier Qty", required=True, default=1)
```

Comment:
Hi. Setting ``required=True and default=1`` + adding ``raise Warning`` prevent users to set custom values, that is a big limitation, and make the module working only for product with a ``Unit`` UoM.
Simply removing both parts will make the module generic. 

(and so also removing ``set_default_multiplier_qty_supplierinfo``)

Reply:
@victoralmau I think it's better to add a boolean field instead saying "Only multipliers". If so, the quantity indicated in the "Min. qty" field will be the multiplier.

Reply:
Hi @pedro. I see what you mean, but there are some limitation in your implementation. 
Min qty is also to mention a price associated to a min qty.
Let's see : 
- product beer (75cl)
- supplierinfo 1 : min_qty : 0, multiplier_qty : 6 ; price : 2,40€
- supplierinfo 1 : min_qty : 300, multiplier_qty : 6 ; price : 2,20€

if you add simply a boolean, you remove such valid configuration.

Reply:
OK, I see. Then it's good to have both indeed. Let's go with the suggestions of not being required. If not filled, no multiplier.